src/app/components/input/input.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { InputType } from 'src/app/core/models/input-type';
+
+@Component({
+  selector: 'app-input',
+  templateUrl: './input.component.html',
+  styleUrls: ['./input.component.scss'],
+})
+export class InputComponent {
+  inputType = InputType;
+
+  @Input()
+  type?: number;
+
+  @Output()
+  inputEvent: EventEmitter<string> = new EventEmitter();
```

Comment:
this is more like output value,  output from component, which contains value about 'input tag'